source/connect/connection-options.txt

```diff
@@ -0,0 +1,324 @@
+.. _cpp-connection-options:
+
+==========================
+Specify Connection Options
+==========================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: connection string, URI, server, Atlas, settings, configure
+
+Overview
+--------
+
+This page describes the MongoDB connection and authentication options
+available in the {+driver-short+}. You can configure your connection by
+specifying options in the connection URI or by passing an instance of the
+``mongocxx::options::client`` class as the ``client_options`` parameter to the
+``mongocxx::client`` constructor.
+
+.. note::
+
+   You can specify some connection options only in the connection URI, and others only
+   in an the ``client_options`` parameter.
+   You might need to combine these methods to specify all the options that you need.
+
```

Comment:
S: might be helpful to put the next two sections ("Using the Connection URI" and "Using a ``mongocxx::options::client`` Object") in their own section (called something like "Set Connection Options"), rather than in "Overview." That way, the top-level headers seem a little more complete (Overview, Set Options, Read Options, Connection URI Table)
```suggestion
Set Connection Options
----------------------
```